spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/support/MetricStore.java

```diff
@@ -50,17 +57,24 @@
 	 */
 	public MetricStore(MetricsProperties metricsProperties) {
 		this.metricsProperties = metricsProperties;
-		restTemplate = new RestTemplate();
+		ObjectMapper mapper = new ObjectMapper();
```

Comment:
I will open up another issue for this, need to merge this now since current master isn't working end to end.